operator/bundle/manifests/loki-operator.clusterserviceversion.yaml

```diff
@@ -647,13 +649,19 @@ spec:
                 - --logtostderr=true
                 - --tls-cert-file=/var/run/secrets/serving-cert/tls.crt
                 - --tls-private-key-file=/var/run/secrets/serving-cert/tls.key
-                - --v=2
+                - --v=0
                 image: quay.io/openshift/origin-kube-rbac-proxy:latest
                 name: kube-rbac-proxy
                 ports:
                 - containerPort: 8443
                   name: https
-                resources: {}
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 128Mi
+                  requests:
+                    cpu: 5m
+                    memory: 64Mi
```

Comment:
I can't recall exactly why/when, but we have a silent requirent to skip adding req/limits for operators on OCP.

Reply:
Please re move them